packages/blade/src/components/Spinner/BaseSpinner/SpinnerIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import { Svg, Path } from '~components/Icons/_Svg';
+
+type SpinnerIconProps = {
+  dimensions: string;
+  color: string;
+};
+
+const SpinnerIcon = ({ dimensions, color }: SpinnerIconProps): React.ReactElement => {
```

Comment:
Don't we want to add this in `icons/`?

Reply:
imo let's keep it here. Specific to spinner only. No need for consumers to expose this.

Reply:
Yeah. Don't see a use-case for consumers to be using this icon standalone